enterprise_access/apps/customer_billing/management/commands/mark_stalled_checkout_intents.py

```diff
@@ -0,0 +1,151 @@
+            threshold_time = timezone.now() - timedelta(seconds=threshold_seconds)
+            stalled_intents = CheckoutIntent.objects.filter(
+                state=CheckoutIntentState.PAID,
+                modified__lte=threshold_time,
+            ).order_by('modified')
+
+            count = stalled_intents.count()
+
+            if count == 0:
+                self.stdout.write(
+                    self.style.SUCCESS(
+                        f'{mode_label}No stalled CheckoutIntent records found.'
+                    )
+                )
+                logger.info('%sNo stalled CheckoutIntent records found', mode_label)
+            else:
+                self.stdout.write(
+                    self.style.WARNING(
+                        f'{mode_label}Found {count} stalled CheckoutIntent record(s):'
+                    )
+                )
+                for intent in stalled_intents:
+                    time_stalled = (timezone.now() - intent.modified).total_seconds()
+                    self.stdout.write(
+                        f'  - ID: {intent.pk}, '
+                        f'User: {intent.user.email if intent.user else "N/A"}, '
+                        f'Enterprise: {intent.enterprise_name or intent.enterprise_slug}, '
+                        f'Time stalled: {int(time_stalled)}s, '
+                        f'Last modified: {intent.modified.isoformat()}'
+                    )
+                    logger.info(
+                        '%sWould mark CheckoutIntent %s as stalled (stalled for %s seconds)',
+                        mode_label,
+                        intent.pk,
+                        int(time_stalled),
+                    )
+        else:
+            # Actually update records
+            updated_count, updated_uuids = CheckoutIntent.mark_stalled_fulfillment_intents(
+                stalled_threshold_seconds=threshold_seconds
+            )
+
+            if updated_count == 0:
+                self.stdout.write(
+                    self.style.SUCCESS(
+                        'No stalled CheckoutIntent records found.'
+                    )
+                )
+                logger.info('No stalled CheckoutIntent records found')
+            else:
+                self.stdout.write(
+                    self.style.SUCCESS(
+                        f'Successfully marked {updated_count} CheckoutIntent record(s) '
+                        f'as errored_fulfillment_stalled'
+                    )
+                )
+                for intent_id in updated_uuids:
+                    self.stdout.write(f'  - Updated CheckoutIntent: {intent_id}')
+                    logger.info(
+                        'Marked CheckoutIntent %s as errored_fulfillment_stalled',
+                        intent_id,
+                    )
+
+        self.stdout.write(
+            self.style.SUCCESS(
+                f'{mode_label}Command completed successfully'
+            )
+        )
```

Comment:
IMO, this is the only self.stdout.write() I'd keep, to leverage the success style.